lib/dream_html.ml

```diff
@@ -865,3 +864,101 @@ module Hx = struct
   let ws_connect fmt = string_attr "data-ws-connect" fmt
   let ws_send = attr "data-ws-send"
 end
+
+module MathML = struct
+  (* Attributes *)
+  let accent = bool_attr "accent"
+  let accentunder = bool_attr "accentunder"
+  let columnspan = int_attr "columnspan"
+  let depth fmt = string_attr "depth" fmt
+
+  let display value =
+    ( "display",
+      match value with
+      | `block -> "block"
+      | `inline -> "inline" )
+
+  let displaystyle = bool_attr "displaystyle"
+  let fence = bool_attr "fence"
+  let framespacing fmt = string_attr "framespacing" fmt
+  let height fmt = string_attr "height" fmt
+  let linethickness fmt = string_attr "linethickness" fmt
+  let lspace fmt = string_attr "lspace" fmt
+  let maxsize fmt = string_attr "maxsize" fmt
+  let minsize fmt = string_attr "minsize" fmt
+  let movablelimit = bool_attr "movablelimit"
+
+  let notation value =
+    ( "notation",
+      match value with
+      | `actuarial -> "actuarial"
+      | `bottom -> "bottom"
+      | `box -> "box"
+      | `circle -> "circle"
+      | `downdiagonalstrike -> "downdiagonalstrike"
+      | `left -> "left"
+      | `longdiv -> "longdiv"
+      | `madruwb -> "madruwb"
+      | `phasorangle -> "phasorangle"
+      | `radical -> "radical"
+      | `right -> "right"
+      | `roundedbox -> "roundedbox"
+      | `top -> "top"
+      | `updiagonalarrow -> "updiagonalarrow"
+      | `updiagonalstrike -> "updiagonalstrike"
+      | `verticalstrike -> "verticalstrike" )
+
+  let rowlines value =
```

Comment:
This is also non-standard.

Reply:
@kentookura thanks for all the work! This is pretty much ready and I want to tweak a couple of minor things like version numbers before merging. I also would like to remove `rowlines` as it is also non-standard: https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mtable#rowlines